priority-queue/PriorityQueueSender/PriorityQueueSender.csproj

```diff
@@ -0,0 +1,19 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <AzureFunctionsVersion>v3</AzureFunctionsVersion>
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="Microsoft.NET.Sdk.Functions" Version="3.0.13" />
```

Comment:
Same as the Low/High, let's just move this to .net 6 on v4 to make sure we're as current as possible.

Reply:
Completed.

Reply:
Done !